html/js/darkMode.js

```diff
@@ -0,0 +1,30 @@
+var stylesheet = document.styleSheets[3]; // dark-mode stylesheet
+
+let darkMode = localStorage.getItem('darkMode');
+const darkToggle = document.getElementById("switch");
+
+const enableDarkMode = () => {
```

Comment:
Do you have an idea about browser compatibility? We don't run this JS through a transpiler, so using `const` and lambda expressions could limit dark mode to some modern browsers.

Reply:
[Const](https://caniuse.com/#feat=const) should be fine. [Arrow functions](https://caniuse.com/#feat=arrow-functions) are not compatible with IE and Opera Mini. I guess that's ok